### PR TITLE
Fixed RPC request serialization issue

### PR DESCRIPTION
**Pull request by lukachi**

### CODE REVIEW

File: packages/connector/src/methods.ts [hunk 1]
```diff
@@ -16,13 +16,16 @@ import type {
...
+  request: { method: string; params?: any },
   snapId: string,
 ): Promise<T> => {
   return await window.ethereum.request({
     method: 'wallet_invokeSnap',
     params: {
-      request,
+      request: {
+        method: request.method,
+        ...(request.params && { params: request.params }),
```
Comment (ihordiachenko) [inline packages/connector/src/methods.ts:27]:
```ts
...(request.params && { params: request.params }),
```

What does it actually do? Adds a `params` only if it exists in the `request` object?

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 [Unreleased]
 
+## [2.1.1] - 2024-06-18
+
+### Fixed
+- `@rarimo/rarime-connector` - request params
```
Comment (ihordiachenko) [inline CHANGELOG.md]: Add a proper description of what was fixed, including the MM version that introduced the change

File: packages/connector/src/methods.ts (hunk 1 above)
Comment (lukachi) [reply packages/connector/src/methods.ts:27]: yes

Comment (ihordiachenko) [reply packages/connector/src/methods.ts:27]: Very hard to read. I'd rather add an explicit `if`

**Pull request merged**